### PR TITLE
Add phyloFlash

### PR DESCRIPTION
**Pull request by epruesse**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
The tests seems to fail here.

File: recipes/phyloflash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  patches:
+    - real_bin.patch
+
+build:
+  number: 0
+
+requirements:
+  build: []
+  run:
+    - perl >=5.13.2
```
Comment (bgruening) [inline recipes/phyloflash/meta.yaml]: You want to have here `r-base` as well.

File: recipes/phyloflash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - r-gtable
+    - r-reshape2
+    - r-ggplot2
+    - r-optparse
+
+test:
+  commands:
+    - (phyloFlash.pl -check_env 2>&1; true) | grep "Please provide a path using -dbhome"
+    - phyloFlash_makedb.pl --help
+    - (phyloFlash_heatmap.R --help; true) | grep "Generates a heatmap plot"
```
Comment (bgruening) [inline recipes/phyloflash/meta.yaml]: The `true` is usually not needed. The return code that is evaluated is taken from the grep command and not from the first command.

Comment (epruesse) [inline recipes/phyloflash/meta.yaml]: ... unless `pipefail` is set. It's not, in this case, so you are right. I was grasping at straws, trying to figure out why the tests are failing on travis.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**